packages/@cdk-cosmos/core/src/cosmos.ts

```diff
@@ -48,15 +58,15 @@ export class CosmosStack extends Stack implements Cosmos {
     this.Version = getPackageVersion();
 
     this.CdkRepo = new Repository(this, 'CdkRepo', {
-      repositoryName: `core-cdk-repo`,
+      repositoryName: RESOLVE(PATTERN.SINGLETON_COSMOS, 'Cdk-Repo', this).toLowerCase(),
     });
 
     this.RootZone = new HostedZone(this, 'RootZone', {
       zoneName: `${rootZone}.${tld}`.toLowerCase(),
     });
 
     this.CdkMasterRole = new Role(this, 'CdkMasterRole', {
-      roleName: 'Core-CdkMaster-Role',
+      roleName: RESOLVE(PATTERN.SINGLETON_COSMOS, 'CdkMaster-Role', this),
```

Comment:
my only thought is for development we might actually change the pattern dynamically so that we can use the only account for develop on, but have multiple cores (dev mode). So if we dont use the pattern then we can not do things like this.

Reply:
Cool then as discussed we can just fix the arn to use this pattern as well on line :
```
this.CdkMasterRoleStaticArn = `arn:aws:iam::${this.account}:role/Core-CdkMaster-Role`;
```